src/nvim/charset.c

```diff
@@ -882,10 +879,12 @@ bool vim_iswordp(char_u *p)
 bool vim_iswordp_buf(char_u *p, buf_T *buf)
   FUNC_ATTR_PURE FUNC_ATTR_WARN_UNUSED_RESULT FUNC_ATTR_NONNULL_ALL
 {
+  int c = *p;
+
   if (MB_BYTE2LEN(*p) > 1) {
```

Comment:
Why not use `c` here like in the upstream patch?

Reply:
I missed that. I thought it passed `p`, not `*p`. I'll fix it.